### PR TITLE
Add coin

### PR DESCRIPTION
**Pull request by pokeylope**
Change "hat" parameter to "type" and add "text" parameter.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,26 +55,41 @@ const text = [
...
 function onLoad() {
-    const hatType = (new URLSearchParams(window.location.search)).get('hat');
-    if (hatType === 'chef') {
-        const floaty = document.querySelector(".floaty");
-        if (!floaty.src.includes('squid-hat.png')) {
-            floaty.src = 'squid-hat.png';
-            floaty.alt = "A giant squid, adorned with a chef's hat, floating ominously on the page";
-            return;
+    const params = new URLSearchParams(window.location.search)
+    const type = params.get('type');
```
Comment (dannybd) [inline index.js]: For backwards compatibility (so the `?hat=chef` links work), can we make this:
```
    const params = new URLSearchParams(window.location.search);
    const type = params.get('type') || params.get('hat');
```

**Review by dannybd — CHANGES_REQUESTED** [review]
This looks good! One change so we don't break existing links, and I think we're good to go :)

File: index.js (hunk 1 above)
Comment (pokeylope) [reply index.js]: Done.

**Comment by dannybd** [comment]
Thanks!

**Pull request merged**